_pages/production.md

```diff
@@ -2,81 +2,152 @@
-  - text: Staging environment
-    href: "#staging-environment"
-  - text: Production configuration
-    href: "#production-configuration"
+  - text: Confirm Interagency Agreement (IAA)
+    href: "#confirm-interagency-agreement-iaa"
   - text: Request deployment
     href: "#request-deployment"
+  - text: Staging environment
+    href: "#staging-environment"
   - text: Changes to production applications
     href: "#changes-to-production-applications"
 saml_year: 2022
 ---
+<div class="usa-alert usa-alert--warning">
+<div class="usa-alert__body">
+<p class="usa-alert__text"><b> Please note: The deployment process will take 1 to 2 weeks to complete </b></p>
+</div>
+</div>
 
-## Production endpoints
+Before deployment
+-----------------
 
-Our integration documentation includes endpoint urls for our sandbox environment <https://idp.int.identitysandbox.gov/>. Our production environment is located at <https://secure.login.gov/>. The URL path to each endpoint remains the same. Only the domain will change. For example, the authorization endpoint will change as follows:
-* OpenID Connect: <https://secure.login.gov/openid_connect/authorize>
-* SAML: <https://secure.login.gov/api/saml/auth{{ page.saml_year }}>
+Make sure you have the following items ready before you start the deployment process:
 
-Please be aware that the IDP certificate (X509 Certificate) in the production environment is different from the IDP certificate in the sandbox environment. The production IDP certificates can be found here:
-* OpenID Connect: <https://secure.login.gov/api/openid_connect/certs>
-* SAML: <https://secure.login.gov/api/saml/metadata{{ page.saml_year }}>
+-   [Signed Interagency Agreement (IAA) listing this integration ](https://developers.login.gov/production/#confirm-iaa)
 
-## Deployments
+-   Authority To Operate (ATO) approved environment
 
-All changes to integrations between Login.gov and your application must be reviewed and deployed. **We ask for at least 2 weeks notice for new integrations and changes to existing integrations. Push Notification URLs may require 3 weeks notice in order to allow the domain for outbound communication.** Regular deployments occur every Thursday by the close of the business day. If the regular deployment occurs on a holiday, then it will be completed the following Monday.
+-   [New app on the dashboard ](https://developers.login.gov/production/#production-configuration-process)
+
+-   Approved logo for your app
+
+-   A [Login.gov production account](https://secure.login.gov)
+
+**Please note: The following items are required to launch your app to production:**
+
+-   All production urls should have .gov, .mil, or a dedicated .com address and point to an ATO-ed environment.
+
+-   You must include a logo for your application. [Learn more about our logo guidelines](https://developers.login.gov/design-guidelines/#agency-logo-guidelines).
+
+-   If this is a SAML integration (Not OpenID Connect), then please ensure that:
```

Comment:
The three points under this need to be nested under it since they're related to SAML integrations.